openmdao/core/problem.py

```diff
@@ -1972,7 +2020,8 @@ def _write_var_info_table(self, header, col_names, meta, vals, print_arrays=Fals
             rows.append(row)
 
         col_space = ' ' * col_spacing
-        print(add_border(header, '-'))
+        out_stream.write(add_border(header, '-'))
```

Comment:
Instead of replacing the `print` call with two `write` calls, you could pass a `file=out_stream` arg to print. That way you don't need a separate call to add a newline.

Reply:
changed the `write` calls into `print` with `file=out_stream`